gui/components/context_menu.py

```diff
@@ -28,3 +28,7 @@ def select_delete_account_from_context_menu(self):
     @allure.step('Select edit account option from context menu')
     def select_edit_account_from_context_menu(self):
         self._context_edit_account_option.click()
+
+    @allure.step('Verify delete option present in context menu')
```

Comment:
Maybe change step name to Get delete option visibility in context menu? Or smth like this

Reply:
absolutely, fixed